.gitignore

```diff
@@ -1,5 +1,7 @@
+/.idea/
 /.php-cs-fixer.php
 /.php-cs-fixer.cache
+/.phpunit.cache
```

Comment:
As this changed in 10? Why not use the already ignored code `.phpunit.result.cache`?

Reply:
done: https://github.com/arderyp/panther/commit/7124b9e85133b04cda0ead906b47024f56c0fea4

Reply:
yes phpunit to changed from `.phpunit.result.cache` to `.phpunit.cache'.  The [9.5 documentation](https://docs.phpunit.de/en/9.5/configuration.html) mentions a `cacheResultFile` property that can be defined in `phpunit.xml` to set a custom cache file, but I do not see it in the [10 documentation](https://docs.phpunit.de/en/10.0/configuration.html), so it may be deprecated.